main

```diff
@@ -0,0 +1,46 @@
+require_relative 'Teacher'
+require_relative 'Student'
+require_relative 'Corrector'
+require_relative 'Classroom'
+require_relative 'Book'
+require_relative 'Rental'
+require_relative 'modules'
+# Books
+# rubocop:disable Metrics/CyclomaticComplexity
+def mainmenu(bookload, studentsload, teachersload, rentalsload)
+  all_books = bookload
+  all_students = studentsload
+  all_teachers = teachersload
+  all_rentals = rentalsload
+  message = [' ', 'Welcome to School Library App!', ' ', 'Please choose an option by entering a number:',
+             '1 - List all books', '2 - List all people', '3 - Create a person', '4 - Create a book',
+             '5 - Create a rental', '6 - List all rentals for a given person id', '7 - Exit']
+  message.each { |e| puts e }
+  option = gets.chomp
+  case option
+  when '1'
+    list_books(all_books)
+    mainmenu(all_books, all_students, all_teachers, all_rentals)
+  when '2'
+    list_people(all_students, all_teachers)
+    mainmenu(all_books, all_students, all_teachers, all_rentals)
+  when '3'
+    returned = add_person(all_students, all_teachers)
+    mainmenu(all_books, returned[0], returned[1], all_rentals)
+  when '4'
+    mainmenu(add_book(all_books), all_students, all_teachers, all_rentals)
+  when '5'
+    result = create_rental(all_books, all_students, all_teachers, all_rentals)
+    mainmenu(all_books, all_students, all_teachers, result)
```

Comment:
- [ ] When trying to create rentals, I uncovered some issues:
  - A person is currently unable to rent more than one book.
  - A book cannot be rented to more than one person.
  
Below is the error message I get:

![image](https://user-images.githubusercontent.com/16058470/136643922-1785c77a-a9e1-47d6-bfb8-2f6129fa7a14.png)

Kindly fix it.